packages/headless/src/features/search/search-actions-thunk-processor.ts

```diff
@@ -121,11 +121,12 @@ export class AsyncSearchThunkProcessor<RejectionType> {
 
   public async fetchFromAPI(
     {mappings, request}: MappedSearchRequest,
-    origin: SearchOrigin
+    origin: SearchOrigin,
+    disableAbortWarning?: boolean
```

Comment:
I'm worried that we may end up having even more optional paramters in the future. Do you think it would make sense to extract `origin` and `disableAbortWarning` into an object?